bigquery_etl/metadata/parse_metadata.py

```diff
@@ -166,6 +166,11 @@ def validate_labels(self, attribute, value):
                             raise ValueError(f"Invalid label format: {bug}")
                 else:
                     raise ValueError("Error: review_bugs needs to be a list.")
+            elif re.match(r"owner\d", key):
+                if not is_email(label):
+                    raise ValueError(
```

Comment:
Should we skip Github identities instead of raising an exception? E.g.
if not is_email_or_github_identity(label):